docs/chapter2-payload-image-format.rst

```diff
@@ -0,0 +1,659 @@
+                                              since the payload can access any part of the
+                                              FIT without needing this property to indicate
+                                              which images are needed.
+   compatible        O           string list  List of compatible strings for Platform Init
+                                              to use when selecting the best configuration,
+                                              in order from most specific / desirable to
+                                              least. This may be used to support using the
+                                              same FIT with many different boards, where
+                                              Platform Init understands the concept of a
+                                              compatible string and can make this
+                                              determination.
+   require-fit       O           boolean      If this property exists, it means the whole
+                                              fit image shall be loaded together before
+                                              Platform Init calling payload entry.
+   ================= =========== ============ ==============================================
+
+
+2.3.1 Example FIT
+~~~~~~~~~~~~~~~~~
+
+Shown below is a FIT structure in source form (Image Tree Source). It shows
+three images and a single configuration.
+
+.. code-block:: none
+
+   / {
+    	description = “Uefi Payload”
+    	timestamp = <0x00000000>
+    	#address-cells = <0x02>;
+    	size = <0x00385000>	
+    	spec-version = <0x00000100>;
+    	build-revision = <0x00010105>;
+    	images {
+    		tianocore {
+    			description = "Tianocore edk2-stable202208";
+    			timestamp = <0x00000000>
+    			project = "tianocore";
+    			arch = "x86_64";
+    			type = "flat-binary";
+    			capabilities = "smm-rebase", "...";
+    			producer = "My company";
+    			data-offset = <...data…>;
+    			data-size = <...data…>;
+    			reloc-start = <start offset of reloc table within data>;
+    			entry-start = <0x121b10>;
+    			load = <0x120000>;
+    		};
+    		uefi-fv {    // showing how to have multiple images
+    	description = "UEFI Firmware Volume";
+    	type = "flat-binary";
+    			arch = "x86_64";
+    			project = "tianocore";
+    			compression = "lzma";
+    			
+    			data-offset = <...data…>;
+    			data-size = <...data…>;
+    		};
+    		bds-fv {
+    			description = "BDS Firmware Volume";
+    			type = "flat-binary";
+    			arch = "x86_64";
+    			project = "tianocore";
+    		compression = "lzma";
+    			
+    		data-offset = <...data…>;
+    		data-size = <...data…>;
+    	};
+    };
+    	configurations {
+    		default = "conf-1";
+    		conf-1 {
+    			firmware = "tianocore"
+    			require-fit;
+    		};
+    	};
+    };
+
+
+This shows a Tianocore ELF file as the Payload, with a signature to protect the
```

Comment:
I didn't see any ELF related info above. Currently universal UEFI payload uses ELF format, late we might change it to PE/COFF format. Better to move it from this spec not to mislead users late.

Reply:
Done reworded. Could you help to check again?

Reply:
New change looks to me.